app/src/main/res/values/colors.xml

```diff
@@ -4,11 +4,13 @@
     <color name="colorIsCurrentUserNotSick">#4CAF50</color>
 
     <!-- Imported from UX Repo -->
-    <color name="colorPrimary">#BE3E49</color>
-    <color name="colorSecondary">#FFFFFF</color>
+    <color name="colorPrimary">#2c58b1</color>
```

Comment:
James added named color specs, I think we should source `colorPrimary` from those (I think you'll see it when merging with `develop`)

Reply:
Ok, just overwrite my change when merging.